tools/gyp/pylib/gyp/generator/make.py

```diff
@@ -800,7 +800,7 @@ def Write(self, qualified_target, base_path, output_filename, spec, configs,
           gyp.xcode_emulation.MacPrefixHeader(
               self.xcode_settings, lambda p: Sourceify(self.Absolutify(p)),
               self.Pchify))
-      sources = filter(Compilable, all_sources)
+      sources = list(filter(Compilable, all_sources))
```

Comment:
sources = [s for s in all_sources if Compilable(i)]

Reply:
the changes in this commit are just copied from `node-gyp`. do you think it's ok to deviate?

Reply:
Its ok to deviate for functional reasons, but in this case, I'm not sure if we should. @cclauss is that just more idiomatic code, or is it required for py3+py2 support? if so, should it be in both repos?

Reply:
Idiomatic only — easier for mortals to understand.  We can leave it as is.